helpers/trasladosHelper/trasladosHelper.go

```diff
@@ -182,11 +182,12 @@ func RegistrarTraslado(data *models.Movimiento) (result *models.Movimiento, outp
 	}
 
 	// Crea registro en api movimientos_arka_crud
-	if res, err := crudMovimientosArka.PostMovimiento(data); err != nil {
+	if err := crudMovimientosArka.PostMovimiento(data); err != nil {
```

Comment:
Revisar: Quizás falte esto ... ?
```suggestion
	if err := crudMovimientosArka.PostMovimiento(&data); err != nil {
```